agent/xds/testdata/endpoints/transparent-proxy-with-peered-upstreams.latest.golden

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
Same as the previous comment. I would expect the db upstreams endpoints to be in here.